src/modules/dailyCreditsSummary/i18n/sv_SE.ts

```diff
@@ -102,5 +102,6 @@ export default {
             raw: 'JSON (raw)',
             prettified: 'JSON (prettified)',
         },
+        csv: 'CSV',
```

Comment:
Since CSV is the abbreviation for comma seperated values the name shouldn't change in different languages. So could be static

Reply:
I decided to add them to allow translators to add additional information and also "JSON" is in the translations too.

But we can remove it, if it disturbes too much ;)

Reply:
I am with @Cr4zyc4k3 on this one. Since both JSON and CSV are abbreviations they don't need localization.

Reply:
The translations of `JSON` and `CSV` have been removed in f1e8f72.